src/mozillavpn.h

```diff
@@ -126,6 +126,7 @@ class MozillaVPN final : public QObject {
   Q_PROPERTY(bool debugMode READ debugMode CONSTANT)
   Q_PROPERTY(QString currentView READ currentView WRITE setCurrentView NOTIFY
                  currentViewChanged)
+  Q_PROPERTY(QObject* releaseMonitor READ releaseMonitor CONSTANT)
```

Comment:
Don't do this. Expose the releaseMonitor as VPNReleaseMonitor in `commandui.cpp`